requirements/edx/development.txt

```diff
@@ -348,7 +348,7 @@ django-filter==2.4.0
     #   -r requirements/edx/testing.txt
     #   edx-enterprise
     #   lti-consumer-xblock
-django-ipware==3.0.7
+django-ipware==4.0.0
```

Comment:
This file also includes the changes not necessarily relevant to this pull request. I don't think it's a good idea to blend them.

Reply:
Since we are updating one dependency it will result in a cascade of other dependency updates. 

That said, a lot of the changes you are commenting on are from the previous PR: https://github.com/edx/edx-platform/pull/28475

Once that is merged to master I will resume work on this.